business_requirement_deliverable_project/models/project.py

```diff
@@ -11,6 +11,39 @@ class Project(models.Model):
 
     origin = fields.Char('Source Document')
 
+    business_requirement_id = fields.Many2one(
+        'business.requirement',
+        string='Business Requirement',
+        help='Link the Project and the business requirement',
+    )
+
+    business_requirement_deliverable_id = fields.Many2one(
+        comodel_name='business.requirement.deliverable',
+        string='Business Requirement Deliverable',
+        help='Link the Project and the business requirement deliverable',
+    )
+
+    @api.multi
+    def open_linked_br_dl(self):
```

Comment:
@darshan-serpent This returns type action so the name need to be according:
s/open_linked_br_dl
r/action_open_linked_br_dl